DepthImages/obstacle_detection.py

```diff
@@ -17,6 +17,8 @@ def detect(input, width, threshold):
     array = array[array != 0]
 
     # For now we assume super close object mostly filling screen = bad.
+    # The following 2 lines return 1 if there are too many error states in the array, 1 means nothing happens in test_haptics 
+    # so if we want to be over cautious we should probably make far away buzz
```

Comment:
It might make it a bit confusing to the user if we have too many buzzers. What I was thinking was at this point we would only have one handle vibrate if an obstacle was coming and whichever handle vibrates the user has to move in that direction